docs/products/kafka/kafka-connect/getting-started.rst

```diff
@@ -0,0 +1,41 @@
+Getting started
+===============
+
+Aiven services are managed in the Aiven `web console <https://console.aiven.io/>`__ . When you first log in to the console with your email address and password, you will see the **Services** view, which shows you all the services in the currently selected project.
```

Comment:
What if we make it more relatable for the reader and replace "Aiven services are managed in the Aiven " with "You can manage the Aiven services in our web console"

Reply:
I took this piece from the main Aiven for Apache Kafka, not sure if we should change both